rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +96,35 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
```

Comment:
Do you want to make this "special considerations for the use of origin `openconfig` and origin `cli`"? This seems to be the really specific case that we're handling.

Reply:
Will do